packages/gnome_nibbles.rb

```diff
@@ -1,45 +1,34 @@
 require 'package'
 
 class Gnome_nibbles < Package
+  @_commit = '62964e9256fcac616109af874dbb2bd8342a9853'
```

Comment:
Can we move this line closer to the `source_url` line?  I'd like to reserve the top for description, homepage, version, license and compatibility.  This also maintains consistency in packages which can help avoid confusion.

Reply:
Done